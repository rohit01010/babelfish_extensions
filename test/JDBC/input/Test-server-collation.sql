SELECT CAST(SERVERPROPERTY('collation') as VARCHAR(50))
GO