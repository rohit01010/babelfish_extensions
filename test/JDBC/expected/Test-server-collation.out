SELECT CAST(SERVERPROPERTY('collation') as VARCHAR(50))
GO
~~START~~
varchar
sql_latin1_general_cp1_ci_as
~~END~~

